Formula/emacs-plus@31.rb

```diff
@@ -267,8 +267,8 @@ def caveats
       Emacs.app was installed to:
         #{prefix}
 
-      To link the application to default Homebrew App location:
-        osascript -e 'tell application "Finder" to make alias file to posix file "#{prefix}/Emacs.app" at posix file "/Applications" with properties {name:"Emacs.app"}'
+      To get Emacs to show up in Spotlight, move the app to the /Applications folder:
+        mv "#{prefix}/Emacs" /Applications/Emacs.app
```

Comment:
shouldn't it be 

```
mv "#{prefix}/Emacs.app" /Applications/Emacs.app
```

?

Reply:
doesn't matter, it doesn't work anyway

Reply:
it would be really nice if this could be fixed. I have had to open emacs by going to Finder, pressing Cmd+a to go to /Applications and typing "Ema" and pressing Cmd+o for the last few months and it's really annoying

Reply:
Look, Emacs+ doesn't provide cask, meaning it can not install any `.app` files in `/Application` or `~/Application`. So for now it's up to the user to deal with that 🤷 

I have just tried, the following works:

```
cp -r /opt/homebrew/opt/emacs-plus@30/Emacs.app ~/Applications/Emacs.app
```

Reply:
> doesn't matter

actually it does matter :) Because the following also works:

```
cp -r /opt/homebrew/opt/emacs-plus@30/Emacs.app /Applications/Emacs.app
```